src/mudlet.pro

```diff
@@ -157,7 +157,7 @@ BUILD = $$(MUDLET_VERSION_BUILD)
 isEmpty( BUILD ) {
 # Leave the value of the following empty for a release build
 # i.e. the line should be "BUILD =" without quotes
```

Comment:
This comment may be outdated, and `BUILD = ""` does indeed work. Let's see if `BUILD =` works, too.

Reply:
`BUILD =` works - which is why I put that comment there!